snippet_ranger/utils.py

```diff
@@ -0,0 +1,97 @@
+from collections import defaultdict
+
+from ast2vec import Source
+from ast2vec.bblfsh_roles import get_role_id
+
+
+def uast_to_bag(uast, bag=None, role="SIMPLE_IDENTIFIER"):
+    """
+    Convert UAST to bag of words for certain role.
+
+    :param uast: Uast to process.
+    :param bag: Specify existing bag of words if you want to update it.
+    :param role: Role or list of roles to get from uast and convert to bag of words.
+    :return: bag of words for tokens of nodes.
+    """
+    if bag is None:
+        bag = defaultdict(int)
+    # (Konst) Can be optimised if we need
```

Comment:
The format is:

`# TODO(<your github id>): blah-blah-blah`

It is the standard. They are parsed by the IDE and collected in a list.